dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +419,35 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+function getSummarizedProgressForAnonymousUser () {
+  var summarizedProgress = {};
+  var levelProgress = {};
+
+  summarizedProgress.lines = $.cookie('lines');
+
+  var allLevelsProgress = JSON.parse(decodeURIComponent($.cookie('progress')));
+  for(var level in allLevelsProgress) {
+    var individualLevelProgressDescription = '';
```

Comment:
Replace individualLevelProgress with levelProgress (and likewise for the description version)

individualLevelProgressDescription is so long I find it hurts rather than helps readability.  Also, I believe we don't currently compile or minify these names, so it bloats the size of the file.  (This is a lesser concern but somewhat relevant.)